ci-operator/step-registry/gather/proxy/gather-proxy-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: gather-proxy
+  from_image:
+    namespace: ocp
+    name: "4.5"
+    tag: upi-installer
+  commands: gather-proxy-commands.sh
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  documentation: |-
+    The pre-deprovision artifacts step collects logs of all containers captured by loki. Even of containers created but no longer presented in the cluster.
```

Comment:
nit: copy/paste error?  Should probably talk about proxy log gathering and deprovisioning.